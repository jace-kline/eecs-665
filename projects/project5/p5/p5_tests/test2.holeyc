bool f(int a) {
    return (a > 0);
}

int main() {
    bool p;
    int x;
    TOCONSOLE p + 1;
    p = (true + 3) * 4;
    p = true && (false || 3);
    p = f("a" * 4);
    x = 1 + p();
    p = (true + false) == 7;
    return 0;
}