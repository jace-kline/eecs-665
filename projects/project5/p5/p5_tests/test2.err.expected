FATAL [8,15]: Arithmetic operator applied to invalid operand
FATAL [9,10]: Arithmetic operator applied to invalid operand
FATAL [10,27]: Logical operator applied to non-bool operand
FATAL [11,11]: Arithmetic operator applied to invalid operand
FATAL [12,13]: Attempt to call a non-function
FATAL [13,10]: Arithmetic operator applied to invalid operand
FATAL [13,17]: Arithmetic operator applied to invalid operand
Type Analysis Failed